bin/worker.js

```diff
@@ -136,8 +136,8 @@ co(function *() {
     config[field] = program[field];
   });
 
-  // If isolated containers is set override capacity...
-  if (config.isolatedContainers) {
+  // If isolated containers is set override capacity (as long as capacity is > 0
+  if (config.isolatedContainers && config.capacity > 0) {
```

Comment:
You want this because hosts sets `capacity = 0` to provoke shutdown... where would it do so before this point?
In respawns? Ahh, when the spot shutdown thing is being advertised and we restart... I don't like that we have that restart, but hey... I guess this is safe...

IMO, we should avoid the respawning, the problem with doing it is that this is somewhat obscure...

I would write a longer comment explaining why you do this.